tensorflow-core/tensorflow-core-api/.bazelversion

```diff
@@ -0,0 +1 @@
+3.7.2
```

Comment:
Is this file required? The version does not match the one we currently use in the CI (3.1.0) so I'm afraid it can cause some sort of conflicts?

Anyway I think I'll merge this after releasing 0.3.0 since that feature is only useful if you compile from sources so no need to have it now.

Reply:
We could (and probably should) change it to what the CI uses.  It's necessary for anyone using bazelisk, which afaik is the recommended way to use bazel, since otherwise it automatically uses the latest bazel version (4.0.0) which isn't supported by `tensorflow` yet.

Optionally we could just gitignore it and leave it to users, but imo matching the CI is better.